.travis.yml

```diff
@@ -7,14 +7,5 @@ rust:
   - 1.26.0
 
 script:
-  - cargo build
-  - cargo build --release
-  - cargo build --no-default-features
-  - cargo build --release --no-default-features
-  - cargo test
-  - cargo test --release
-  - |
-    if [ $TRAVIS_RUST_VERSION == nightly ]; then
-      cargo test --features nightly
-      cargo test --features nightly --release
-    fi
+  - ./ci/script.sh
+  - (cd crossbeam-utils && ./ci/script.sh)
```

Comment:
Well, `ci/script.sh` is pretty much the standard file name for this use (because it belongs to the `script` section in `travis.yml`).

Other examples:

* https://github.com/rust-lang/regex/blob/master/ci/script.sh
* https://github.com/rust-lang-nursery/rust-bindgen/blob/master/ci/script.sh
* https://github.com/rust-lang-nursery/rust-bindgen/blob/master/ci/script.sh
* https://github.com/budziq/rust-skeptic/blob/master/ci/script.sh
* https://github.com/japaric/rust-everywhere/blob/master/ci/script.sh
* https://github.com/BurntSushi/xsv/blob/master/ci/script.sh

Reply:
Ah, okay - sorry, I looked into Travis docs briefly and didn't see it so assumed it was non-standard. It's an odd convention, but better to stick to it than confuse people.